app/screens/reaction_list/reaction_list.js

```diff
@@ -194,28 +191,49 @@ export default class ReactionList extends PureComponent {
                     headerHeight={37.5}
                     theme={theme}
                 >
+                    <View style={[style.emojiAliasContainer, generateTableLandscapePadding(isLandscape)]}>
+                        <Text style={style.emojiAlias}>
+                            {emojiAliases}
+                        </Text>
+                    </View>
                     {this.renderReactionRows()}
                 </SlideUpPanel>
             </View>
         );
     }
 }
 
+const generateTableLandscapePadding = (isLandscape) => {
+    if (isLandscape && DeviceInfo.isTablet()) {
```

Comment:
you could use `DeviceTypes` from constants instead of `DeviceInfo` to avoid having to look through the codebase to change it.